spec/features/user/user_can_connect_to_github_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe "as a logged in user with a github account" do
+  it "can link accounts to connect with github" do
+    OmniAuth.config.mock_auth[:github] = nil
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new({
+      :provider => 'github',
+      :uid => '123545',
+      :credentials => {'token' => ENV['TEST_GITHUB_TOKEN']}
+        })
```

Comment:
This is nice

Reply:
Got the OmniAuth Mock Auth to work – well done!